4.0.0/migration.md

```diff
@@ -27,135 +146,148 @@ The migration process is initiated by the Installer if the following requirement
 
 2. Ensure that PostgreSQL databases, users, and grants are initialized by the `postgres-init-job` job:
 
+        deploy.go:1711] removing postgres db init job
+        deploy.go:1714] running postgres db init job
+        deploy.go:1726] waiting postgres db init job to finish... (attempt 1/10)
+        deploy.go:1730] postgres db init finished correctly
 
-          deploy.go:1711] removing postgres db init job
-          deploy.go:1714] running postgres db init job
-          deploy.go:1726] waiting postgres db init job to finish... (attempt 1/10)
-          deploy.go:1730] postgres db init finished correctly
 
+3. Ensure that collector block for the new agent connections is enabled. If successful, you will see the following:
 
-3. Ensure that collector block for the new agent connections is enabled. To do so, run
+        deploy.go:1560] toggled collector block to true: Accepting new agent connections is OFF
 
-        HTTP GET /api/admin/agents/new/connections?block=true
+   Under the hood, Installer run this HTTP request to API to block new agent connections:
```

Comment:
Under the hood, Installer sends this HTTP request to the API to block new agent connections: